bindings/ergo-lib-wasm/src/ergo_box.rs

```diff
@@ -98,12 +117,13 @@ impl ErgoBox {
         contract: &Contract,
         tx_id: &TxId,
         index: u16,
+        tokens: &Tokens,
```

Comment:
recall: we needed this so that we can build a transaction in WASM and use WASM to calculate the `boxId`. The `from_json` function requires the boxId as a parameter (which there was no way of knowing if for example you're creating boxes for mock data)